packages/adapter/src/chains/oak.ts

```diff
@@ -242,6 +242,7 @@ export class OakAdapter extends ChainAdapter {
       encodedCall,
       encodedCallWeight,
       overallWeight,
+      "PayThroughSovereignAccount",
```

Comment:
adapter should expose the options to caller and not hardcode this parameter. The enum is already defined above, so please make sure this `scheduleXcmpTask()` function include the `instructionSequence` in its parameter list.

```
export enum OakAdapterTransactType {
  PayThroughRemoteDerivativeAccount,
  PayThroughSoverignAccount,
}
```